h2o-webserver-iface/src/main/java/water/webserver/iface/Credentials.java

```diff
@@ -35,7 +36,8 @@ public String toDebugString() {
    */
   private static String base64EncodeToString(String s) {
     final byte[] bytes = s.getBytes(StandardCharsets.ISO_8859_1);
-    return Base64.encodeBase64String(bytes);
+
+    return BASE_64_ENCODING.encode(bytes);
```

Comment:
@Pscheidl this is somewhat risky: `BaseEncoding` is marked unstable, and if they later change it to a thread-unsafe implementation, we can have problems that are hard to find; that's why I chose `commons-codec` which is also already bundled in our distro

Reply:
Ok now I see, commons-codec is a transitive dependency already present there. Good, reverting.